lib/js/modules/z.thumbviewer.js

```diff
@@ -91,11 +91,13 @@ $.widget("ui.thumbviewer",
 
     setWidthHeight: function() {
         this.bigImg.attr({
-            width: $(this.bigImg, this.element.parent()).width(),
-            height: $(this.bigImg, this.element.parent()).height() 
+            // get the original image dimensions,
+            // rather than the calculated ones
+            width: $(this.bigImg, this.element.parent())[0].width,
+            height: $(this.bigImg, this.element.parent())[0].height
```

Comment:
These are the key changes – the dimensions of the image _itself_ need to be collected, rather than the calculated ones which jQuery returns.